src/translate_sections.rs

```diff
@@ -93,7 +93,9 @@ impl binemit::RelocSink for UnimplementedRelocSink {
     ) {
         unimplemented!()
     }
-
+    fn reloc_constant(&mut self, _: binemit::CodeOffset, _: binemit::Reloc, _: ir::ConstantOffset) {
+        unimplemented!()
+    }
```

Comment:
Please add a newline before and after this function.

Reply:
Thanks. Just updated.